### PR TITLE
stub Fast USDC CLI

### PR DESCRIPTION
**Pull request by turadg**
refs: #10328

## Description
Sets up a package for Fast USDC and stubs out its CLI.

Finishing #10328 will require a contract that has the offer makers.

### Security Considerations
none

### Scaling Considerations
none

### Documentation Considerations
This will need end user documentation later

### Testing Considerations
basic test

### Upgrade Considerations
not deployed

### CODE REVIEW

File: packages/fast-usdc/test/cli.test.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import test from 'ava';
+import { execa } from 'execa';
+import { fileURLToPath } from 'url';
+import { dirname, join } from 'path';
+
+const dir = dirname(fileURLToPath(import.meta.url));
+const CLI_PATH = join(dir, '../src/cli.js');
+
+test('CLI shows help when run without arguments', async t => {
+  const { stderr } = await execa('node', [CLI_PATH], {
```
Comment (dckc) [inline packages/fast-usdc/test/cli.test.ts]: `execa` is pretty heavy-weight. I see lots of stuff added to `yarn.lock`. I suspect it won't cost more than 10 lines to stick with node's stdlib. Shall we try it?

**Review by dckc — COMMENTED** [review]
I'd rather not take on execa for this.

**Review by dckc — APPROVED** [review]
¡vamos!

File: packages/fast-usdc/package.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "name": "fast-usdc",
+  "private": true,
+  "version": "0.1.0",
+  "description": "Create an Agoric Javascript smart contract application",
```
Comment (samsiegart) [inline packages/fast-usdc/package.json:5]: This seems generic, is there a better description we can give?

**Review by samsiegart — APPROVED** [review]

File: packages/fast-usdc/package.json (hunk 2 above)
Comment (turadg) [reply packages/fast-usdc/package.json:5]: oops, good catch. that was copied from create-dapp. I'll fix it in a future PR or this one if I need to rev CI

**Pull request merged**